### PR TITLE
RUN-4238: not override remote-debugging-port from ChromeDriver

### PR DESCRIPTION
**Pull request by wenjunche**
Tests

[W10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b4fafd454b21953031f374f)
[W7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b4faf3b54b21953031f374e)

### CODE REVIEW

**Comment by HarsimranSingh** [comment]
Added this ticket to the .34 sprint.

File: index.js [hunk 1]
```diff
@@ -167,8 +167,12 @@ if (coreState.argo['local-startup-url']) {
         let localConfig = JSON.parse(fs.readFileSync(coreState.argo['local-startup-url']));
 
         if (typeof localConfig['devtools_port'] === 'number') {
-            console.log('remote-debugging-port:', localConfig['devtools_port']);
-            app.commandLine.appendSwitch('remote-debugging-port', localConfig['devtools_port'].toString());
+            if (!coreState.argo['remote-debugging-port']) {
+                console.log('remote-debugging-port:', localConfig['devtools_port']);
```
Comment (datamadic) [inline index.js]: I think this needs to be `log.writeToLog('info', ' ... ');` for it to actually show up in the logs no?

**Comment by StevenEBarbaro** [comment]
@HarsimranSingh I believe we're holding on 8.56.30.* patches. We can always cherry-pick and create a PR. Let's make a note in the JIRA

**Comment by datamadic** [comment]
noted in Jira, will merge.  @StevenEBarbaro

**Pull request merged**